src/nspawn/nspawn-network.c

```diff
@@ -503,6 +506,58 @@ int test_network_interfaces_initialized(char **iface_pairs) {
         return 0;
 }
 
+static int netns_fork_child(int netns_fd, int *ret_original_netns_fd) {
```

Comment:
maybe call this netns_child_begin() or so

the reason i am asking for this is because i was confused as first, because this function is after all not the full child, but just the initialization of the child, the main part of the child comes after this function returns after all. maybe a more precise name might help the next reader